downloads/aws/README.md

```diff
@@ -0,0 +1,81 @@
+Scripted setup to get your aws account ready for Spinnaker
+
+Replaces setup at https://www.spinnaker.io/setup/install/providers/aws/
+
+## Prep Work
+These steps must be completed before you can setup your aws account
+
+### Environment
+You can set up AWS from your local machine or from an instance launched in your AWS account.
+You need these permissions to set up AWS to run Spinnaker:
+```
+	ec2:*
+	iam:*
+	sts:passRole
+	sts:AssumeRole
+```
+
+If you also want to run `halyard`, you'll need `s3` permissions:
+```
+	s3:*
+```
+
+AWS Inline Policy Example (no restrictions):
+```json
+{
+    "Statement": [
+        {
+            "Action": [
+                "iam:*",
+                "ec2:*",
+                "s3:*",
+                "sts:PassRole",
+                "sts:AssumeRole"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "*"
+            ]
+        }
+    ],
+    "Version": "2012-10-17"
+}
+```
+A policy giving you access to these resources should be attached to the instance profile you're running this script from. If you've already launched the instance, you can add permissions to the instance profile via the AWS Console.
+
+The permissions listed here are not all of the permissions needed to run Spinnaker. These permissions are just for preparing your aws environment. 
```

Comment:
These permissions are extremely extreme. I think it'd be good to mention that these permissions are strictly for account setup and are the path of least resistance, rather than rule of least privilege, and is not intended or recommended for the operation of Spinnaker itself.

Reply:
I'll add that. Also, this permissions list is somewhat lazy on my part. I'm not sure exactly which permissions you need for all of these things, and I don't want to narrow it down in my first pass because it's not trivial.